app/services/merge_requests/refresh_service.rb

```diff
@@ -87,6 +88,20 @@ def comment_mr_with_commits
       end
     end
 
+    # Call merge request webhook with update branches
+    def execute_mr_web_hooks
+      merge_requests = @project.origin_merge_requests.opened
+                               .where(source_branch: @branch_name)
+                               .to_a
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.